### PR TITLE
Cjw#9/two pointer

### PR DESCRIPTION
**Pull request by cJinu**

### CODE REVIEW

File: src/BS/CJW/BOJ_1208_부분수열의 합.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		for (int i=1; i<(1<<size); i++){
+			long t=0;
+			for (int j=0; j<size; j++){
+				if (i == (i|(1<<j))) t+= arr[j];
+			}
+			if (map.containsKey(t)) map.put(t, map.get(t)+1);
+			else map.put(t, (long)1);
+		}
+		return map;
+	}
```
Comment (cuzzzu1318) [inline src/BS/CJW/BOJ_1208_부분수열의 합.java:38]: 뭔가요 이건....

File: src/TwoPointer/CJW/제출을 위한 문서.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+반갑습니다.
+어느새 추워진 날씨와 재빠르게 지나간 길었던 연휴가 지나간 시간을 떠오르게 합니다.
+
+다들 긴 연휴간 안녕하셨는지요?
+
+저는 책상앞 세상을 벗어나 조금은 더 아름다운 세상을 즐기고 돌아왔습니다.
```
Comment (cuzzzu1318) [inline src/TwoPointer/CJW/제출을 위한 문서.txt:6]: 가끔은 벗어날 필요도 있죠

**Review by cuzzzu1318 — APPROVED** [review]

**Review by thingseong — APPROVED** [review]
나 가을타나봐

File: src/BS/CJW/BOJ_1208_부분수열의 합.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import java.util.*;
+public class Main {
+
+	public static void main(String[] args) throws IOException {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		String[] st = br.readLine().split(" ");
+		int n=Integer.parseInt(st[0]),s=Integer.parseInt(st[1]);
+		int mid = n/2;
+		int[] arr = new int[mid];
+		int[] brr = new int[n-mid];
```
Comment (sukmd) [inline src/BS/CJW/BOJ_1208_부분수열의 합.java:11]: brrrrrrrrrrrrrrrrrrr

**Review by sukmd — APPROVED** [review]

**Review by Jo-dv — APPROVED** [review]

**Pull request merged**